src/precice/impl/SolverInterfaceImpl.hpp

```diff
@@ -518,6 +518,11 @@ class SolverInterfaceImpl {
     return _accessorCommunicatorSize;
   }
 
+  /// Synchronizes all ranks and deduces the total amount changes meshes
```

Comment:
```suggestion
  /// Synchronizes all ranks and deduces the total amount of mesh changes
```
or total amount of changed meshes?